util/git/creds_test.go

```diff
@@ -1,12 +1,16 @@
 package git
 
 import (
+	"fmt"
+	"github.com/argoproj/argo-cd/v2/util/cert"
 	"github.com/argoproj/argo-cd/v2/util/io"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"io/ioutil"
 	"os"
+	"path"
```

Comment:
Sort the imports?

Reply:
Done! I just picked some GoLand settings I liked. Not sure if Argo CD has any opinion about order.